src/gossip/gossip_service.zig

```diff
@@ -173,11 +182,17 @@ pub const GossipService = struct {
 
         var echo_server = echo.Server.init(allocator, my_contact_info.gossip.port(), logger, exit);
 
+        const entrypoint_list = if (entrypoints) |eps| blk: {
```

Comment:
can probably change the logic to be more like
```
entrypoint_list = ArrayList().init(allocator)
if (entrpoints) || { 
    try entrypoint_list.ensureCapacity(len)
    for (..) {  entrypoint_list.appendAssumeCapacity() }
}
```
also should use `appendAssumeCapacity`

Reply:
https://github.com/Syndica/sig/pull/69/commits/5e89a807cd66e9e6fc957f207c2da1ad9f4943db